datasets/miam/README.md

```diff
@@ -0,0 +1,293 @@
+- `Dialogue_ID`: identifier of the dialogue as a string.
+- `File_ID`: identifier of the source file as a string.
+
+For the `ilisten` configuration, the different fields are:
+- `Speaker`: identifier of the speaker as a string.
+- `Utterance`: Utterance as a string.
+- `Dialogue_Act`: Dialog act label of the utterance. It can be one of "AGREE" (0), "ANSWER" (1), "CLOSING" (2), "ENCOURAGE-SORRY" (3), "GENERIC-ANSWER" (4), "INFO-REQUEST" (5), "KIND-ATTITUDE_SMALL-TALK" (6), "OFFER-GIVE-INFO" (7), "OPENING" (8), "PERSUASION-SUGGEST" (9), "QUESTION" (10), "REJECT" (11), "SOLICITATION-REQ_CLARIFICATION" (12), "STATEMENT" (13) or "TALK-ABOUT-SELF" (14).
+- `Dialogue_ID`: identifier of the dialogue as a string.
+
+For the `loria` configuration, the different fields are:
+- `Speaker`: identifier of the speaker as a string.
+- `Utterance`: Utterance as a string.
+- `Dialogue_Act`: Dialog act label of the utterance. It can be one of "ack" (0), "ask" (1), "find_mold" (2), "find_plans" (3), "first_step" (4), "greet" (5), "help" (6), "inform" (7), "inform_engine" (8), "inform_job" (9), "inform_material_space" (10), "informer_conditioner" (11), "informer_decoration" (12), "informer_elcomps" (13), "informer_end_manufacturing" (14), "kindAtt" (15), "manufacturing_reqs" (16), "next_step" (17), "no" (18), "other" (19), "quality_control" (20), "quit" (21), "reqRep" (22), "security_policies" (23), "staff_enterprise" (24), "staff_job" (25), "studies_enterprise" (26), "studies_job" (27), "todo_failure" (28), "todo_irreparable" (29), "yes" (30)
+- `Dialogue_ID`: identifier of the dialogue as a string.
+- `File_ID`: identifier of the source file as a string.
+
+For the `maptask` configuration, the different fields are:
+- `Speaker`: identifier of the speaker as a string.
+- `Utterance`: Utterance as a string.
+- `Dialogue_Act`: Dialog act label of the utterance. It can be one of "acknowledge" (0), "align" (1), "check" (2), "clarify" (3), "explain" (4), "instruct" (5), "query_w" (6), "query_yn" (7), "ready" (8), "reply_n" (9), "reply_w" (10) or "reply_y" (11).
+- `Dialogue_ID`: identifier of the dialogue as a string.
+- `File_ID`: identifier of the source file as a string.
+
+For the `vm2` configuration, the different fields are:
+- `Utterance`: Utterance as a string.
+- `Dialogue_Act: Dialogue act label of the utterance. It can be one of "ACCEPT" (0), "BACKCHANNEL" (1), "BYE" (2), "CLARIFY" (3), "CLOSE" (4), "COMMIT" (5), "CONFIRM" (6), "DEFER" (7), "DELIBERATE" (8), "DEVIATE_SCENARIO" (9), "EXCLUDE" (10), "EXPLAINED_REJECT" (11), "FEEDBACK" (12), "FEEDBACK_NEGATIVE" (13), "FEEDBACK_POSITIVE" (14), "GIVE_REASON" (15), "GREET" (16), "INFORM" (17), "INIT" (18), "INTRODUCE" (19), "NOT_CLASSIFIABLE" (20), "OFFER" (21), "POLITENESS_FORMULA" (22), "REJECT" (23), "REQUEST" (24), "REQUEST_CLARIFY" (25), "REQUEST_COMMENT" (26), "REQUEST_COMMIT" (27), "REQUEST_SUGGEST" (28), "SUGGEST" (29), "THANK" (30).
```

Comment:
```suggestion
- `Dialogue_Act`: Dialogue act label of the utterance. It can be one of "ACCEPT" (0), "BACKCHANNEL" (1), "BYE" (2), "CLARIFY" (3), "CLOSE" (4), "COMMIT" (5), "CONFIRM" (6), "DEFER" (7), "DELIBERATE" (8), "DEVIATE_SCENARIO" (9), "EXCLUDE" (10), "EXPLAINED_REJECT" (11), "FEEDBACK" (12), "FEEDBACK_NEGATIVE" (13), "FEEDBACK_POSITIVE" (14), "GIVE_REASON" (15), "GREET" (16), "INFORM" (17), "INIT" (18), "INTRODUCE" (19), "NOT_CLASSIFIABLE" (20), "OFFER" (21), "POLITENESS_FORMULA" (22), "REJECT" (23), "REQUEST" (24), "REQUEST_CLARIFY" (25), "REQUEST_COMMENT" (26), "REQUEST_COMMIT" (27), "REQUEST_SUGGEST" (28), "SUGGEST" (29), "THANK" (30).
```
missing backtick